Source/GSHorizontalTypesetter.m

```diff
@@ -1035,9 +1034,9 @@ the first tab would move (exactly) to the next tab
 		  g->nominal = NO;
 		p.x += last_p.x;
 		p.y += last_p.y;
+	    last_p = g->pos = p;
 	      }*/
 
-	    last_p = g->pos = p;
```

Comment:
I am not sure whether the setting of `g->pos` that got removed here is really not needed. If so, could you please explain why?

Reply:
You are right, this was too greedy on my side.